packages/client/bin/logging/logtape.ts

```diff
@@ -0,0 +1,89 @@
+import { getFileSink } from '@logtape/file'
+import type { Logger as LogtapeLoggerType } from '@logtape/logtape'
+import {
+  ansiColorFormatter,
+  configure,
+  getConsoleSink,
+  getLogger as getLogtapeLogger,
+} from '@logtape/logtape'
+import type { Logger } from '../../src/types.ts'
+
+const LEVELS: Record<string, number> = {
+  error: 0,
+  warn: 1,
+  info: 2,
+  debug: 3,
+}
+
+export class LogtapeLogger implements Logger {
+  public logger: LogtapeLoggerType
+  private logLevel: string
```

Comment:
I guess the `logLevel` should be kept out here, this is too much `statefulness`, also the log level is already saved in the client, so this would be redundant.

Reply:
Fixed.